python/Cargo.toml

```diff
@@ -30,16 +30,16 @@ edition = "2018"
 libc = "0.2"
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync"] }
 rand = "0.7"
-pyo3 = { version = "0.14.1", features = ["extension-module"] }
-datafusion = { git = "https://github.com/apache/arrow-datafusion.git", rev = "4d61196dee8526998aee7e7bb10ea88422e5f9e1" }
+pyo3 = { version = "0.14.2", features = ["extension-module"] }
+datafusion = { path = "../datafusion", features = ["pyarrow"] }
 
 [lib]
-name = "datafusion"
+name = "internals"
```

Comment:
This isn't a public thing that anyone was referring to by name?

Reply:
Nope, it produces a `datafusion.internals` submodule (which we can also rename using a leading underscore).

Reply:
Renamed it to `_internal`